core/domain/feedback_jobs_continuous_test.py

```diff
@@ -147,16 +147,17 @@ def test_multiple_threads_multiple_exp(self):
             ModifiedFeedbackAnalyticsAggregator.start_computation()
             self.assertEqual(self.count_jobs_in_taskqueue(), 1)
             self.process_and_flush_pending_tasks()
+            feedback_analytics = (
```

Comment:
Seems more prudent to create new tests instead of replacing the ones for get_thread_analytics()? Unless you plan to get rid of the latter altogether (which isn't unreasonable; it could redirect to the get_multi but with just one element).